implentio/ui-toolkit
# TagChip component

created TagChip component
created TagChip story
added TagChip to src/index.ts exports

## Description
This is a chip component used in tables. You can tag things with the tag chip

## Link to JIRA ticket and Figma Design
https://www.figma.com/file/UYBp8YXmCgUqhiOsrkHd6V/Implentio-Library?type=design&node-id=111-339&mode=design&t=REkWM0YRpdpXhUB0-0

## Applicable Screenshots
default
![image](https://github.com/implentio/ui-toolkit/assets/93560211/2e7bd379-1acd-42ac-9162-bd2e30205d79)
active
![image](https://github.com/implentio/ui-toolkit/assets/93560211/78149b07-9bcc-4a6b-96e2-a65cdc569687)
focus
![image](https://github.com/implentio/ui-toolkit/assets/93560211/13450c78-8316-41e2-b485-656b6f1fc896)


## Testing Instructions
- Fetch and checkout my branch `git fetch && git checkout <feat/IUT-44/TagChip>`
- npm run storybook
- check for each variant:

1. default state
2. hover state
3. active state (if any)
4. focus state (if any)

## Checklist:
Check what is applicable by putting an 'x' in between the []:
